package.json

```diff
@@ -20,12 +20,14 @@
     "lib"
   ],
   "main": "lib/index.js",
-  "types": "lib/index",
+  "typings": "lib/index.d.ts",
   "scripts": {
     "clean": "rimraf lib && rimraf coverage",
     "format": "yarn prettier --write '{src,__tests__}/**/*.ts'",
     "lint": "tslint -p . --fix",
     "prebuild": "yarn clean",
+    "prepare": "npm run build",
```

Comment:
@linhlecle I think you should not add this script because we have built before publish already.